content/05.model-complexity.md

```diff
@@ -1,31 +1,32 @@
+In addition to cascade learning, approaches that better represent rare classes using class re-balancing techniques like inverse sampling probability weighting [@doi:10.1186/s12911-021-01688-3], inverse class frequency weighting [@doi:10.1197/jamia.M3095], oversampling of rare classes [@https://doi.org/10.1613/jair.953], or uniformly random undersampling of majority class [@doi:10.48550/arXiv.1608.06048] may also help minimize issues associated with class imbalance.
 
-The presence of label-noise and sparsity in the data can lead to overfitting, meaning that the models show high prediction accuracy on the training data but low prediction accuracy on new evaluation data.
-Overfit models tend to rely on patterns that are unique to the training data (for example, the clinical coding practices at a hospital), and not generalizable to new data (e.g., data collected at different hospitals). [@isbn:0262035618; @pmc:PMC8238368]
+The presence of label-noise and sparsity in the data can lead to poor generalizability or overfitting, meaning that the models show high prediction accuracy on the training data but low prediction accuracy on new evaluation data.
+Overfit models tend to rely on patterns that are unique to the training data, such as the clinical coding practices at a hospital, and not generalize to new data such as data collected at different hospitals. [@isbn:0262035618; @pmc:PMC8238368]
 Regularization approaches can help mitigate these scenarios by adding constraints to a model to avoid making large prediction errors.
-This not only protects ML models and learned representations from poor generalizability caused by overfitting, but also reduces model complexity by decreasing the feature space available for training [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. (Figure[@fig:3]a)
+This protects ML models from poor generalizability by reducing model complexity and minimizing model feature space [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. (Figure[@fig:3]a)
 Examples of ML methods with regularization include ridge regression, LASSO regression, and elastic net regression [@doi:10.1111/j.1467-9868.2005.00503.x], among others.
 Regularization is often used in exploring functional role of variants in rare disease and immune cell signature discovery studies; much like rare disease, these examples need to accommodate sparsity in data.
 For example, LASSO regularization reduced the number of features (genes) included as features in an ML model designed to classify brain tissue regions of amyotrophic lateral sclerosis (ALS) patients. [@doi:10.1186/s10020-023-00603-y]
```

Comment:
Justin: This doesn’t really demonstrate the purpose of LASSO. Would focus on LASSO as performing feature selection and sparse models to aid in interpretability.

Reply:
[In the next line] Justin: What is EN doing that might led to better performance, and what about the underlying data is amenable to EN?

Reply:
```suggestion
LASSO regularization was used to select a few informative genes as features to include in models designed to classify brain tissue samples between amyotrophic lateral sclerosis (ALS) patients and healthy patients with high accuracy, thus making the models more interpretable. [@doi:10.1186/s10020-023-00603-y]
```

Reply:
The edit for line 32 is also included in the above comment